java/transforms-by-suitability/README.md

```diff
@@ -1,84 +1,32 @@
-
-```java
-    SpatialReference inputSr = mOriginalGeometry.getSpatialReference();
-    SpatialReference outputSr = mArcGISMap.getSpatialReference();
-
-    List<DatumTransformation> transformationsBySuitability;
-    if (mUseExtentForSuitability) {
-      transformationsBySuitability = TransformationCatalog.getTransformationsBySuitability(
-          fromSr, toSr, mMapView.getVisibleArea().getExtent());
-
-    } else {
-      transformationsBySuitability = TransformationCatalog.getTransformationsBySuitability(inputSr, outputSr);
-    }
-
-    DatumTransformation defaultTransform = TransformationCatalog.getTransformation(inputSr, outputSr);
-```
-
-When the user taps on a transformation in the list, the `OnItemClickListener` responds by using the selected transformation to reproject a `Point` and then adds the point as a `Graphic`. If the selected transformation is not usable (has missing grid files) then an exception is thrown.
-
-```java
-    // Get the datum transformation selected by the user
-    DatumTransformation selectedTransform = (DatumTransformation)adapterView.getAdapter().getItem(i);
-
-    Point projectedGeometry = null;
-    try {
-      // Use the selected transformation to reproject the Geometry
-      projectedGeometry = (Point) GeometryEngine.project(mOriginalGeometry, mMapView.getSpatialReference(),
-          selectedTransform);
-
-      } catch (ArcGISRuntimeException agsEx) {
-        // Catch errors thrown from project method. If a transformation is missing grid files, then it cannot be
-        // successfully used to project a geometry, and will throw an exception.
-        Snackbar.make(tableList, agsEx.getMessage() + "\n" +
-            getResources().getString(R.string.transform_missing_files), Snackbar.LENGTH_LONG).show();
-        removeProjectedGeometryGraphic();
-        return;
-    }
-```
+
+1. Pass the input and output spatial references to `TransformationCatalog.getTransformationsBySuitability` for transformations based on the map's spatial reference OR additionally provide an extent argument to only return transformations suitable to the extent. This returns a list of ranked transformations.
+2. Use one of the `DatumTransformation` objects returned to project the input geometry to the output spatial reference.
 
 ## Relevant API
-* TransformationCatalog
+
 * DatumTransformation
 * GeographicTransformation
 * GeographicTransformationStep
```

Comment:
it looks like our sample doesn't use either of these classes

Reply:
> it looks like our sample doesn't use either of these classes

they are used in the DatumTransformationAdapter.java file.

Reply:
ah whoops, I missed that. thanks!